raiden/ui/app.py

```diff
@@ -181,12 +181,16 @@ def run_app(
         api_port = Port(DEFAULT_HTTP_SERVER_PORT)
 
     # Store the flat fee settings for the given token networks
+    # The given flat fee is for the whole mediation, but that includes two channels.
+    # Therefore divide by 2 here.
     token_network_to_flat_fee: Dict[TokenNetworkAddress, FeeAmount] = {
-        address: fee for address, fee in flat_fee
+        address: FeeAmount(fee // 2) for address, fee in flat_fee
     }
+    # Given prop. fee also counts per mediation, therefore divide by 2 as well
+    # Division is fine here, as the prop. fees are applied independently, not subsequently
     fee_config = MediationFeeConfig(
         token_network_to_flat_fee=token_network_to_flat_fee,
-        proportional_fee=proportional_fee,
+        proportional_fee=ProportionalFeeAmount(proportional_fee // 2),
```

Comment:
This would need to be `1 - sqrt(1 - proportional_fee)`then, if I did the calculation right.

Reply:
According to my calculation it's `sqrt(proportional_fee + 1) - 1`: https://www.wolframalpha.com/input/?i=q%3Dp%5E2%2B2p+solve+p

Reply:
Using `u := cli_prop_fee`, `p := new_prop_fee` and `x := payment amount` I get
```
xp + (x - xp) * p = ux
p + (1 - p) * p = u
p + p - p^2 = u
2p - p^2 = u
```
Which differs from your input to alpha.

Reply:
Your're right, my math skill suck 😒